packages/forms-core/src/updaters/value-updater.ts

```diff
@@ -84,7 +84,7 @@ export function ValueUpdaterFactory(helpers: UpdateStoreHelpers): ValueUpdater {
 
             fieldState.data.currentValue = newValue;
             fieldState.data.transientValue = transientValue;
-            formsLogger.log("Setting new selection to:", Object.assign({}, newSelection));
+            formsLogger("Setting new selection to:", Object.assign({}, newSelection));

```

Comment:
Scope the logger.
https://github.com/reactway/forms/blob/39c563f43571aa30ac06352e4f1ad28376afc260/packages/forms/src/modifiers/thousands-modifier.tsx#L4